tests/integration_tests/test_chainermn.py

```diff
@@ -298,9 +298,6 @@ def test_suggest_float(storage_mode: str, comm: CommunicatorBase) -> None:
 
                 assert x1 == x2
 
-                # Adding a parameter should not cause any errors.
-                mn_trial.suggest_float("x1", low1, high1, log=True)
-
```

Comment:
How about testing for an error? It's changed by introducing `FloatDistribution`. https://github.com/optuna/optuna/commit/d5904abb2a6431e79140e3f1fbc36adc419b6986#diff-c61e26fb5e8d21683854b515c6ed466e629b7ea3d155e3846147d0c7ab786825

Reply:
Thank you for the careful review, you're absolutely right.
I add(restore) tests in 3a2a5e8.